lib/rules/validate-jsdoc/require-description-complete-sentence.js

```diff
@@ -56,6 +56,8 @@ function requireDescriptionCompleteSentence(node, err) {
         .replace(/(`)([^`]+)\1/, quotedSanitizer)
         .replace(/(')([^']+)\1/, quotedSanitizer)
         .replace(/(")([^"]+)\1/, quotedSanitizer)
+        .replace(/<([^ >]+)[^>]*>[\s\S]*?.*?<\/\1>|<[^\/]+\/>/g, htmlSanitizer)
+        .replace(/<([^ >]+)[^>]*>/g, htmlSanitizer)
```

Comment:
I don't really like these regexps, but since html in jsdoc isn't complex, I'd say ok.